src/components/page-environment/environment-overview.js

```diff
@@ -163,17 +163,16 @@ export class EnvironmentOverview extends React.Component {
                         </Typography>
                         <Typography>
                           Active deployment{' '}
-                          <Typography link as="span">
-                            <NavLink
-                              className="env__link"
-                              to={routing.getAppDeploymentUrl(
-                                appName,
-                                deployment.name
-                              )}
-                            >
+                          <Link
+                            to={routing.getAppDeploymentUrl(
+                              appName,
+                              deployment.name
+                            )}
+                          >
+                            <Typography link as="span">
                               {smallDeploymentName(deployment.name)}
-                            </NavLink>
-                          </Typography>{' '}
+                            </Typography>{' '}
+                          </Link>
```

Comment:
add the space after the `Link` component instead of inside it. 
```suggestion
                            </Typography>
                          </Link>{' '}
```